VERSION

```diff
@@ -1 +1,2 @@
-0.1.0
+# This is the runtime version.
```

Comment:
Please no comment lines. Just the version. The Makefile relies on it.

Reply:
@bergwolf  removed comments.